### DISCUSSION TITLE
Discussion: Sort: new is incomprehensible

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#101751] [Mobile]
GitHub Mobile beta for Android
![Screenshot_20240123-152827.png](https://github.com/community/community/assets/2119212/2d27f69f-9848-424d-8e84-8575fc0a6561)

I want to sort by creation date, either oldest to newest or newest to oldest. I don't want to sort by last update. 

Pictured: GitHub community/community discussion for category: mobile with the sort "filter" "dropped down". Note that it's sufficiently offscreen that I guessed there was no support for sorting at all. 

There's apparently something, but I don't understand what or why. 

In a browser, there's a `Date created` option:
https://github.com/orgs/community/discussions/categories/mobile
![Screenshot_20240123-153307.png](https://github.com/community/community/assets/2119212/e97f65f2-2588-4629-b289-ba960e10d4d6)

**Comment by nadaayoujil** [comment]
Currently, GitHub Mobile does not support sorting by creation date directly through the interface, and it seems that the option might be offscreen or not easily accessible.

In a browser, you can find the "Date created" option for sorting, but this feature is not yet available in the mobile app :(
